calibrators/tag_based_pnp_calibrator/README.md

```diff
@@ -0,0 +1,123 @@
+| `calibration_crossvalidation_training_ratio`  | `double`              | `0.7`                            | The ratio of data used for training versus validation during the calibration's cross-validation process.                                                                                                                                                                |
+| `calibration_convergence_min_pairs`           | `int`                 | `9`                              | The minimum number of apriltag and lidartag detection pairs required to consider the calibration process as potentially converged.                                                                                                                                      |
+| `calibration_convergence_min_area_percentage` | `double`              | `0.005`                          | Minimum percentage of the area that needs to be covered by detection.                                                                                                                                                                                                   |
+| `min_pnp_points`                              | `int`                 | `8`                              | Minimum number of points required for the Perspective-n-Point problem used in calibration to solve the pose estimation.                                                                                                                                                 |
+| `min_convergence_time`                        | `double`              | `6.0`                            | Minimum time required for the active hypotheses to be considered as converged.                                                                                                                                                                                          |
+| `max_no_observation_time`                     | `double`              | `3.0`                            | Hypotheses can timeout when a new observation arrives and the time difference (dt) exceeds the `max_no_observation_time` threshold for the same tag ID, or when the time difference between the latest observation and the earlier observation surpasses the threshold. |
+| `new_hypothesis_distance`                     | `double`              | `1.5`                            | Distance threshold with respect to converged hypotheses for creating a new hypothesis.                                                                                                                                                                                  |
```

Comment:
units

Reply:
fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/82ad48e1f8bb36b7637da582a69974a9f1c145aa